apps/ehr/src/index.tsx

```diff
@@ -39,7 +39,15 @@ root.render(
         }}
         cacheLocation="localstorage"
       >
-        <ErrorBoundary fallback={<p>An error has occurred</p>}>
+        <ErrorBoundary
+          onError={(error, errorInfo) => {
+            console.log(String(error), errorInfo);
+            if (String(error).includes('TypeError: Failed to fetch dynamically imported module')) {
+              location.reload();
```

Comment:
Is a hard reload safe to do? I'm thinking of unsaved edits or embedded workflows